### PR TITLE
BXMSPROD-1601 [main] provide FDB github action for RHBA project

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1601

* https://github.com/kiegroup/droolsjbpm-knowledge/pull/585
* https://github.com/kiegroup/drools/pull/4084
* https://github.com/kiegroup/optaplanner/pull/1749
* https://github.com/kiegroup/optaweb-employee-rostering/pull/744
* https://github.com/kiegroup/optaweb-vehicle-routing/pull/628

### CODE REVIEW

**Comment by yurloc** [comment]
@Ginxo does `.github/pull_request_template.md` need an update too (as in #627)?

**Comment by Ginxo** [comment]
done! many thanks @yurloc (either for the rest of `BXMSPROD-1601_main` branch PR)

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ changes that span multiple kiegroup repositories and depend on each other. -->
...
- 
-[build-chain tool](https://github.com/kiegroup/github-action-build-chain) is a build tool which can be used on command line locally or in Github Actions workflow(s), in case you need to change multiple repositories and send multiple dependent pull requests related with a change you can easily reproduce the same build by executing it on Github hosted environment or locally in your development environment. See [local execution](https://github.com/kiegroup/github-action-build-chain#local-execution) details to get more information about it.
+We do "simple" maven builds, they are just basically maven commands, but just because we have multiple repositories related between them and one change could affect several of those projects by multiple pull requests, we use [build-chain tool](https://github.com/kiegroup/github-action-build-chain) to handle cross repository builds and be sure that we always use latest version of the code for each repository.
```
Comment (yurloc) [inline .github/pull_request_template.md:29]: @Ginxo I think the "Build Chain tool does" version is an improved version of "We do" so let's keep the newer one if possible.

Comment (Ginxo) [reply .github/pull_request_template.md]: done!! thanks @yurloc

**Review by yurloc — APPROVED** [review]

File: .github/workflows/full-downstream.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      matrix:
+        os: [ubuntu-latest]
+        java-version: [11]
+        maven-version: ['3.8.1']
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    name: Full downstream build
+    steps:
+      - name: Clean Disk Space
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/ubuntu-disk-space@main
```
Comment (Ginxo) [inline .github/workflows/full-downstream.yml:38]: I just noticed this is wrong, they should point to kogito-pipelines ones

**Review by yurloc — CHANGES_REQUESTED** [review]
```diff
- uses: kiegroup/droolsjbpm-build-bootstrap/...
+ uses: kiegroup/kogito-pipelines/...
```

**Comment by Ginxo** [comment]
@yurloc changed to kogito-pipelines. I also added `paths-ignore` to pull-request file. Thanks

**Review by yurloc — APPROVED** [review]
LGTM.

**Pull request merged**